crates/ruff_linter/src/rules/pydocstyle/snapshots/ruff_linter__rules__pydocstyle__tests__D208_D.py.snap

```diff
@@ -1,14 +1,13 @@
 ---
 source: crates/ruff_linter/src/rules/pydocstyle/mod.rs
-snapshot_kind: text
 ---
 D.py:252:1: D208 [*] Docstring is over-indented
     |
 250 |     """Summary.
-251 | 
+251 |
+    | ^ D208
 252 |        Description.
```

Comment:
Same here. I think the diagnostic should be at the beginning of the `Description` line